omnigan/trainer.py

```diff
@@ -836,6 +836,7 @@ def get_masker_loss(self, multi_domain_batch):  # TODO update docstrings
                                     prediction,
                                     self.domain_labels["s"],
                                     self.D["s"]["Advent"],
+                                    gan_type=self.opts.dis.s.gan_type,
```

Comment:
This cannot work, gan_type should be provided at init time not call time

Reply:
Yes. I gave up testing because of some network issues last night. I am about to fix that.